packages/firebase_admob/README.md

```diff
@@ -10,45 +10,71 @@ available yet and testing has been limited.
 [Pull Requests](https://github.com/flutter/plugins/pulls) are welcome.
 
 ## Initializing the plugin
+The AdMob plugin must be initialized with an AdMob App ID.
 
-Before any other methods are invoked, the plugin must be initialized with an
-AdMob App ID:
 ```
 FirebaseAdMob.instance.initialize(appId: appId);
 ```
 
 ## Using banners and interstitials
+Banner and interstitial ads can be configured with target information. 
+And in the example below, the ads are given test ad unit IDs for a quick start.
 
-Banner and interstitial ads must be created with an AdMob ad unit ID, and they
-can include targeting information:
 ```
 MobileAdTargetingInfo targetingInfo = new MobileAdTargetingInfo(
-  keywords: <String>['foo', 'bar'],
-  contentUrl: 'http://foo.com/bar.html',
+  keywords: <String>['flutterio', 'beautiful apps'],
+  contentUrl: 'https://flutterio.com',
```

Comment:
That should be `https://flutter.io`?

Reply:
oops, I'll schedule to get back to that after work. :)